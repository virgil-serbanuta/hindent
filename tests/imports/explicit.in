-- Explicit imports - capitals first (typeclasses/types), then operators, then identifiers

import qualified MegaModule as M ((>>>), MonadBaseControl, void, MaybeT(..), join, Maybe(Nothing, Just), liftIO, Either, (<<<), Monad(return, (>>=), (>>)))
