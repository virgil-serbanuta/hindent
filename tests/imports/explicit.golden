-- Explicit imports - capitals first (typeclasses/types), then operators, then identifiers
import qualified MegaModule as M (Either, Maybe(Just, Nothing), MaybeT(..),
                                  Monad((>>), (>>=), return), MonadBaseControl,
                                  (<<<), (>>>), join, liftIO, void)
